ci/vm/provision2.sh

```diff
@@ -59,11 +39,32 @@ xpmem_load() {
   fi
 }
 
+install_gcc() {
+  # Find the GCC ver used to compile the running kernel
+  GCC_VER=$(awk -F ' ' '{print $7}' /proc/version | cut -d'-' -f1)
+  echo "INFO: Installing GCC ver $GCC_VER"
+  wget -q https://ftp.gnu.org/gnu/gcc/gcc-"$GCC_VER"/gcc-"$GCC_VER".tar.xz
```

Comment:
any need to also download/validate signature using the public key?

Reply:
Added, thanks!